src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
box-sizing: border-box; try to add this property
to normilize.css

*{
box-sizing: border-box;
}